portal-web/test/functional/com/liferay/portalweb/macros/SignIn.macro

```diff
@@ -39,6 +39,26 @@ definition {
 		AssertElementNotPresent(locator1 = "Button#I_AGREE");
 	}
 
+	macro portalInstanceSetUp {
+		var portalURL = PropsUtil.get("portal.url");
+		var portalURL = StringUtil.replace("${portalURL}", "http://", "");
+		var portalURL = StringUtil.replace("${portalURL}", ":8080", "");
+
+		var testName = selenium.getTestName();
+
+		var customActivationMessage = "Virtual Instance is ready for ${testName}";
```

Comment:
@kyle-miho No `.`?

Reply:
@vicnate5 i commited the wrong change, i fixed it

Reply:
nevermind i recommited the wrong thing again

Reply:
fixed